app/controllers/posts_controller.rb

```diff
@@ -15,12 +15,11 @@ def new
   end
 
```

Comment:
- [ ] In your `[blog_application](https://github.com/osmanwako/blog_application/tree/integration_specs)/[test](https://github.com/osmanwako/blog_application/tree/integration_specs/test)/[system](https://github.com/osmanwako/blog_application/tree/integration_specs/test/system)
/posts_test.rb` you are missing under the` Post Index Page` a test that redirects to the post's show page when you click on a post.